packages/react/src/icons/IconSort.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { withIcon } from "./withIcon";
+
+interface SortIconProps {
+  sortDirection: "asc" | "desc" | false;
```

Comment:
this doesn't look right

we should have 3 separate icons exported - `IconSort`, `IconSortUp`, and `IconSortDown` and conditionally use that in the component rather than have this logic remain within the icon component

Reply:
Gotcha vai, will update it.